README.rst

```diff
@@ -270,6 +270,28 @@ automated builds of different OpenWISP modules.
 ``openwisp-utils-qa-checks``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Runs all quality assurance checks in a single file
+
+The following quality assurance checks are ran in openwisp-utils-qa-checks:
+
+* ``checkmigrations`` (Read more below)
+* ``checkcommit`` (Read more below)
```

Comment:
This [line](https://github.com/openwisp/openwisp-utils/blob/master/tests/test_project/tests/test_qa.py#L105) "Simulation of a special unplanned case\n\n#noqa" shows an undocumented change. Please document `#noqa` case.

Reply:
@atb00ker Could you explain what you mean here? I believe the `#noqa` case is documented above?

Reply:
Can you please send the link?